src/java.base/share/classes/java/lang/reflect/code/op/CoreOps.java

```diff
@@ -2331,7 +2521,12 @@ protected BinaryOp(BinaryOp that, CopyContext cc) {
         }
 
         protected BinaryOp(String name, TypeDesc resultType, Value lhs, Value rhs) {
```

Comment:
Unused parameter `resultType`. Remove this and it will follow on to the sub-types and to the factory methods.